playbooks/roles/cloudflare/tasks/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
 - name: load vars
   local_action: include_vars conf.yml
+  when:
+    (cf_zone is undefined) or
+    (cf_token is undefined) or
+    (cf_zone is undefined)
 
```

Comment:
please remove this commit from this PR, this is already in the other one. I'll merge it after this.